x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/rules_list/components/notify_badge/notify_badge.test.tsx

```diff
@@ -101,31 +94,26 @@ describe('RulesListNotifyBadge', () => {
       />
     );
 
-    // Open the popover
-    wrapper.find(EuiButtonIcon).first().simulate('click');
-
-    // Snooze for 1 hour
-    wrapper.find('button[data-test-subj="linkSnooze1h"]').first().simulate('click');
-    expect(snoozeRule).toHaveBeenCalledWith({
-      duration: 3600000,
-      id: null,
-      rRule: {
-        count: 1,
-        dtstart: '1990-01-01T05:00:00.000Z',
-        tzid: 'America/New_York',
-      },
-    });
-
-    await act(async () => {
-      jest.runOnlyPendingTimers();
+    userEvent.click(screen.getByTestId('rulesListNotifyBadge-unsnoozed'));
```

Comment:
I think all these should now be
```suggestion
    await userEvent.click(screen.getByTestId('rulesListNotifyBadge-unsnoozed'));
```

Since the user-event upgrade